src/zimsplit.cpp

```diff
@@ -159,40 +163,43 @@ class ZimSplitter
     }
 };
 
+static const char USAGE[] = R"(
+    zimsplit splits smartly a ZIM file in smaller parts.
+
+Usage:
+    zimsplit [--prefix=PREFIX] [--force] --size=N <file>
+    zimsplit --version
+
+Options:
+    --prefix=PREFIX    Prefix of output file parts
+    --size=N            The file size for each part
+    --force             Create zim parts even if it is impossible to have all part size smaller than requested
+    -h, --help          Show this help message
+    --version           Show zimsplit version.
+)";
+
+
 int main(int argc, char* argv[])
 {
   try
   {
-    zim::Arg<std::string> out_prefix(argc, argv, 'o');
-    zim::Arg<int> part_size(argc, argv, 's');
-    zim::Arg<bool> force(argc, argv, "--force");
-    zim::Arg<bool> printVersion(argc, argv, "--version");
-
-    // version number
-    if (printVersion)
-    {
-      version();
-      return 0;
-    }
 
-    if (argc <= 1)
-    {
-      std::cerr << "\nzimsplit splits smartly a ZIM file in smaller parts.\n\n"
-        "usage: " << argv[0] << " [options] zimfile\n"
-        "\n"
-        "options:\n"
-        "  -o            prefix of output file parts\n"
-        "  -s            size of each file parts\n"
-        "  --force       create zim parts even if it is impossible to have all part size smaller than requested\n"
-        "  --version     print the software version\n"
-        << std::flush;
-      return -1;
-    }
+    std::string versionstr("zimsplit " + std::string(VERSION));
+    std::map<std::string, docopt::value> args = docopt::docopt(USAGE,
+                                                            {argv + 1, argv + argc},
+                                                            true,
+                                                            versionstr);
+
+    std::string prefix = PARTPREFIX;
```

Comment:
I would use the file input as default prefix.
This way, a `zimsplit foo.zim` will generate `foo.zimaa`, `foo.zimab`, ...